docx/bookmark.py

```diff
@@ -0,0 +1,11 @@
+# encoding: utf-8
+
+"""Objects related to bookmarks."""
+
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+
+class Bookmarks(object):
+    """Sequence of |Bookmark| objects."""
```

Comment:
Each commit should have a test (occasionally two) and the code to make it pass. And, all tests should pass on every commit (although @wip acceptance tests are skipped until they pass and don't count in the "all-tests" condition). Only the code required to pass the test is included in the commit.

Accordingly, there is no separate commit to add this module. It's added in its minimum form when first required by a test. It was needed in this commit so it could be mocked (or whatever) so it appears here. Note that it has no dependencies, in particular it inherits from `object` here; subclassing `Sequence` will be added when we need its sequence behaviors for a subsequent test.